Formula/borgbackup.rb

```diff
@@ -0,0 +1,32 @@
+class Borgbackup < Formula
+  include Language::Python::Virtualenv
+
+  desc "Deduplicating archiver with compression and authenticated encryption"
+  homepage "https://borgbackup.readthedocs.io/en/stable/"
+  url "https://github.com/borgbackup/borg/releases/download/1.1.13/borgbackup-1.1.13.tar.gz"
+  sha256 "164a8666a61071ce2fa6c60627c7646f12e3a8e74cd38f046be72f5ea91b3821"
+  license "BSD-3-Clause"
+
+  depends_on "pkg-config" => :build
+  depends_on "openssl@1.1"
+  depends_on :osxfuse
+  depends_on "python@3.8"
+
+  resource "msgpack" do
+    url "https://files.pythonhosted.org/packages/74/0a/de673c1c987f5779b65ef69052331ec0b0ebd22958bda77a8284be831964/msgpack-0.6.2.tar.gz"
+    sha256 "ea3c2f859346fcd55fc46e96885301d9c2f7a36d453f5d8f2967840efa1e1830"
+  end
```

Comment:
Guess that does not work, 0.5.6 would be the best working version.

But, also from setup.py (1.1-maint branch):
```
# prefer_system_msgpack is another option, but you need to set it in src/borg/helpers.py.
```

Not sure how stuff is handled by brew. Can it deal with conflicting msgpack version requirements, like one package wanting the latest and borg wanting 0.5.6?

If not, using the msgpack that is bundled into borg might be less troublesome.

https://github.com/borgbackup/borg/blob/1.1.13/src/borg/helpers.py#L34

Reply:
Removed msgpack, so it uses the bundled version.